ImageGallery/ImageGalleryTableViewController.swift

```diff
@@ -26,6 +29,16 @@ class ImageGalleryTableViewController: UITableViewController, UISplitViewControl
         singleTapGesture.require(toFail: doubleTapGesture)
         
         tableView.allowsSelection = false
+        
+        // load data from disk
+        ImageGalleyGlobalDataSource.shared.openGalleriesMapData()
```

Comment:
I would make the call to `openGalleriesMapData()` in the `ImageGalleyGlobalDataSource` constructor